tests/test_workspace_semantic_release_decision.py

```
@@ -0,0 +1,1071 @@
+        / REPO_UUID
+        / "semantic-release-decisions"
+        / GENERATION_ID
+    )
+    decision_generation.mkdir(parents=True, mode=0o700)
+    decision_generation.parent.chmod(0o700)
+    for index in range(65):
+        path = decision_generation / f"{index:064x}.json"
+        path.write_bytes(b"{}")
+        path.chmod(0o600)
+    before = tree_snapshot(harness.state_root)
+    with pytest.raises(CapacityExceeded, match="maximum|64"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)